Endjin.CodeOps/README.md

```diff
@@ -0,0 +1,3 @@
+# Endjin.CodeOps
+
+This PowerShell module contains a variety functions useful for manipulating git repos and .NET solutions.
```

Comment:
Can we have a few more details!

Is there a license?

Reply:
The repo has the standard license - module metadata is now updated and I've added a license badge